packages/blade/tsconfig.json

```diff
@@ -10,6 +10,14 @@
       "~tokens/*": ["src/tokens/*"],
       "~src/*": ["src/*"]
     },
+    "types": [
```

Comment:
why do we have to define it like this? the import from package would resolve these types no?

Reply:
As part of their [TS](https://webdriver.io/docs/typescript/) setup.

webdriver.io injects globals into the tests so we need these types to come up and modify the `expect` etc functions.